openshift/e2e-tests-openshift.sh

```diff
@@ -322,6 +322,19 @@ function tag_built_image() {
   oc tag --insecure=${INSECURE} -n ${EVENTING_NAMESPACE} ${OPENSHIFT_REGISTRY}/${OPENSHIFT_BUILD_NAMESPACE}/stable:${remote_name} ${local_name}:latest
 }
 
+function run_origin_e2e() {
+  curl -L https://github.com/openshift/origin/tarball/master > origin.tar.gz
+  tar xvzf origin.tar.gz
+  cd $(find . -mindepth 1 -maxdepth 1 -name openshift-origin-*)
+
+  # Build tests
+  make build WHAT=cmd/openshift-tests
+
+  # Run tests
+  _output/local/bin/linux/amd64/openshift-tests run all \
```

Comment:
Similar to https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L172 from the `e2e` template.